client/src/scenes/Home/SyncRequestForm/components/ColorPicker.tsx

```diff
@@ -20,59 +21,67 @@ const useStyles = makeStyles(() =>
-        { id: 3, name: 'Grape', color: '#8e24aa' },
-        { id: 4, name: 'Flamingo', color: '#e67c73' },
-        { id: 5, name: 'Banana', color: '#f6c026' },
-        { id: 6, name: 'Tangerine', color: '#f5511d' },
-        { id: 7, name: 'Peacock', color: '#039be5' },
-        { id: 8, name: 'Graphite', color: '#616161' },
-        { id: 9, name: 'Blueberry', color: '#3f51b5' },
-        { id: 10, name: 'Basil', color: '#0b8043' },
-        { id: 11, name: 'Tomato', color: '#d60000' },
-    ];
+    const colors: Color = {
+        '1': { name: 'Lavender', color: '#7986cb' },
+        '2': { name: 'Sage', color: '#33b679' },
+        '3': { name: 'Grape', color: '#8e24aa' },
+        '4': { name: 'Flamingo', color: '#e67c73' },
+        '5': { name: 'Banana', color: '#f6c026' },
+        '6': { name: 'Tangerine', color: '#f5511d' },
+        '7': { name: 'Peacock', color: '#039be5' },
+        '8': { name: 'Graphite', color: '#616161' },
+        '9': { name: 'Blueberry', color: '#3f51b5' },
+        '10': { name: 'Basil', color: '#0b8043' },
+        '11': { name: 'Tomato', color: '#d60000' },
+
+        get '0'() {
+            if (defaultColor === undefined) {
+                return { name: 'Default', color: '#ffffff' };
+            }
+            return { name: 'Default', color: defaultColor };
+        },
+    };
 
     return (
         <div>
-            <FormControl className={style.formControl}>
+            <FormControl className={style.formControl} disabled={defaultColor === undefined}>
                 <InputLabel id="demo-simple-select-label">Color</InputLabel>
                 <Select
                     labelId="demo-simple-select-label"
                     id="demo-simple-select"
                     defaultValue={initialColorId}
                     onChange={(e) => onChange(e.target.value as number)}
                     renderValue={(id) => {
-                        const color = colors.find((color) => color.id === id);
+                        const color = colors[(id as number).toString()];
                         return (
                             <div className={style.renderValueContainer}>
-                                {color?.name}
                                 <div className={style.colorPreview} style={{ backgroundColor: color?.color }} />
+                                {color?.name}
                             </div>
                         );
                     }}
                 >
-                    {colors.map((color) => (
-                        <MenuItem key={color.id} value={color.id}>
-                            {color.name}
+                    {Object.entries(colors).map(([id, color]: [string, { name: string; color: string }]) => (
+                        <MenuItem key={id} value={id}>
                             <div className={style.colorPreview} style={{ backgroundColor: color.color }} />
+                            {color.name}
```

Comment:
Does the switch

Name color to color name is volontary?
It breaks the alignment with the label

Reply:
I invert both in label and in selector to have all color div in aligned at left

Reply:
Okay, I guess it looks better overall :)